check_sites.py

```diff
@@ -36,9 +36,12 @@ def find_diff_sets(old_sets, new_sets):
              for primary, fps in new_sets.items()
              if fps != old_sets.get(primary)
             }
-
     subtracted_sets = {primary: old_sets[primary] for primary in set(old_sets) - set(new_sets)}
-
+    # Make sure that subtracted sets aren't just sets with changed primaries
+    for _, fps in diff_sets.items():
+        for primary in subtracted_sets:
+            if fps.includes(primary):
+                subtracted_sets -= primary
```

Comment:
Can you write a test that would fail if we didn't have this code?

Reply:
In case it isn't clear, [this test](https://github.com/GoogleChrome/first-party-sets/pull/59/files#diff-c40416e23022656073da5094bfddfab89ac7e41d556203fd1968b7135308f365R872) that I added after your comment is meant to address the case.